main.py

```
@@ -18,47 +18,37 @@
+    fred = Fred(api_key=fred_api_key)
 
-bucket = get_gcp_bucket()
-print('Retrieved GCP bucket: {}'.format(bucket))
+    PUSH_TO_GITHUB = os.environ.get("PUSH_TO_GITHUB") != 'False'
+    PUSH_TO_GCP = os.environ.get("PUSH_TO_GCP") != 'False'
+    SAVE_AS_PICKLE = os.environ.get("SAVE_AS_PICKLE") != 'False'
 
-data_map_dict = config["data_map_dict"]
-col_date = config["col_date"]
+    bucket = get_gcp_bucket() if PUSH_TO_GCP else None
+    logger.info('Retrieved GCP bucket: {}'.format(bucket))
```

Ruff (0.8.2)

Use f-string instead of `format` call

Convert to f-string

(UP032)